helm/ph-ee-test/values.yaml

```diff
@@ -318,18 +319,58 @@ ph-ee-engine:
 
   mpesa:
     enabled: false
+    
 
   roster_connector:
     enabled: false
+    
 
   paygops_connector:
     enabled: false
+    
 
   notifications:
     enabled: false
+    
 
   connector_bulk:
-    enabled: false
+    enabled: true
+    image: "419830066942.dkr.ecr.ap-south-1.amazonaws.com/phee-bulk-processor"
+    SPRING_PROFILES_ACTIVE: "bb"
+    imageTag: latest
+    tenant: "rhino,gorilla"
+    hostname: "bulk-connector.sandbox.fynarfin.io"
+    aws:
+      region: "us-east-2"
+      access_key: "AKIAX32JM37TZOJ5AKFB"
+      secret_key: "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"
+    operations_app:
+      contactpoint: "https://ops-bk.sandbox.fynarfin.io/"
```

Comment:
A lot of these contactpoints and hostnames are not matching with integration test hostnames / contact points.
Ex:- Integration test config https://ops-bk2.sandbox.fynarfin.io is not matching with contact point & hostname https://ops-bk.sandbox.fynarfin.io. Keep https://ops-bk2.sandbox.fynarfin.io consistent.